drive/src/fee_pools/epochs/paths.rs

```diff
@@ -27,3 +30,26 @@ impl Epoch {
         vec![vec![RootTree::Pools as u8], self.key.to_vec()]
     }
 }
+
+pub fn encode_key(index: u16) -> Result<[u8; 2], Error> {
```

Comment:
name is wrong.

Reply:
encode_epoch_index or you can put this on the Epoch as a struct method and call it with ::

Reply:
Renamed to `encode_epoch_index_key`